contracts/pm/mixins/MixinContractRegistry.sol

```diff
@@ -1,9 +1,12 @@
-pragma solidity ^0.5.11;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.8;
 
 import "../../ManagerProxyTarget.sol";
-import "./interfaces/MContractRegistry.sol";
+import "../../bonding/IBondingManager.sol";
+import "../../token/IMinter.sol";
+import "../../rounds/IRoundsManager.sol";
 
-contract MixinContractRegistry is MContractRegistry, ManagerProxyTarget {
+contract MixinContractRegistry is ManagerProxyTarget {
```

Comment:
If we're keeping around MContractRegistry anyway could we just inherit from that abstract contract which already imports the interfaces above which would reduce the diff here?